src/lib/QueryRendererHelper.ts

```diff
@@ -75,9 +75,15 @@ export function getQueryForQueryRenderer(source: string, globalQuery: GlobalQuer
 export type HappensDate = keyof Pick<Task, 'scheduledDate' | 'dueDate'>;
 
 export function getDateFieldToPostpone(task: Task): HappensDate | null {
-    const scheduledDateOrNull = task.scheduledDate ? 'scheduledDate' : null;
-    const dateTypeToUpdate = task.dueDate ? 'dueDate' : scheduledDateOrNull;
-    return dateTypeToUpdate;
+    if (task.dueDate) {
+        return 'dueDate';
+    }
+
+    if (task.scheduledDate) {
+        return 'scheduledDate';
+    }
+
+    return null;
```

Comment:
@claremacrae I know that this might be a "too quick" for one change. I could have gone a longer route with expanding the `?:`'s to `if-else`'s. However the logic here is so simple that is was just easier to write this code. I tested broke the code on each line and the tests have failed so we have 100% code coverage here.

Reply:
Huh - I could have sworn I replied to this - it seems the comment got lost?

It's fine - covered by tests - and when I saw the previous code I immediately thought of exactly this simpler implementation...